webapp/src/app/user/review-activity-card/review-activity-card.component.html

```diff
@@ -11,10 +11,16 @@
 } @else {
   <a hlmCard variant="profile" class="hover:bg-accent/50" [href]="htmlUrl() ?? this.pullRequest()?.htmlUrl" target="_blank" rel="noopener noreferrer" [class]="class()">
     <div hlmCardContent variant="profile">
-      <div class="text-sm text-muted-foreground first-letter:uppercase">
-        {{ relativeActivityTime() }}
-        in
-        {{ this.repositoryName() }} #{{ this.pullRequest()?.number }}
+      <div class="flex items-center justify-between">
+        <span class="text-sm text-muted-foreground first-letter:uppercase">
+          {{ relativeActivityTime() }}
+          in
+          {{ this.repositoryName() }} #{{ this.pullRequest()?.number }}
+        </span>
+        <div class="flex items-center text-violet-500 dark:text-violet-600">
```

Comment:
Can you add a tooltip? To me it was not immediately obvious that this is the score.

Reply:
Done in 01b1fa3.